aiida/work/workchain.py

```diff
@@ -2,14 +2,19 @@
 
```

Comment:
Can we add a couple of paragraphs of documentation to how calling old wfs from new ones is expected to be used? Since this will eventually disappear when old workflows are deprecated, I would add this as a short section to the "old workflows" documentation http://aiida-core.readthedocs.io/en/latest/workflow/index.html